deepmd/pt/model/task/dos.py

```diff
@@ -96,6 +105,63 @@ def output_def(self) -> FittingOutputDef:
             ]
         )
 
+    def compute_output_stats(
```

Comment:
the output stat should be able to merge with the `change_out_bias` provided by  #3480.

Reply:
I can update this once #3480 is merged if we want to wait, or fix in a separate PR later.